src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -376,6 +376,10 @@ private void OnBatchScopeDisposed()
                             return s.RemoveAdditionalDocument(id);
                         });
 
+                    // Update project's order of documents if we can.
```

Comment:
what does "if we can" mean in this context?

Reply:
There may not be source files to order in the batch, I think that is what I was referring to.